pallets/staking/src/lib.rs

```diff
@@ -3167,6 +3203,34 @@ impl<T: Trait> Module<T> {
             );
             let rest = max_payout.saturating_sub(validator_payout);
 
+            // Schedule Rewards for the validators
+            let next_block_no = <frame_system::Module<T>>::block_number() + 1.into();
+            for (index, validator_id) in T::SessionInterface::validators().into_iter().enumerate() {
+                let schedule_block_no = next_block_no + index.saturated_into::<T::BlockNumber>();
+                match T::RewardScheduler::schedule(
+                    DispatchTime::At(schedule_block_no),
+                    None,
+                    HIGHEST_PRIORITY,
+                    RawOrigin::Root.into(),
+                    Call::<T>::payout_stakers_by_system(validator_id.clone(), active_era.index).into()
+                ) {
+                    Ok(_) => log!(
+                        info,
+                        "💸 Rewards are successfully scheduled for validator id: {:?} at block number: {:?}",
+                        &validator_id,
+                        schedule_block_no,
+                    ),
+                    Err(e) => {
+                        log!(
+                            error,
+                            "💸 Detected error in scheduling the reward payment: {:?}",
```

Comment:
"Flying money" symbol :smile: 
Other suggestions: ☠ or ⛔ or ⚠
What does Substrate use for the error icon?

Reply:
I will go for this ⚠ as it is a kind of warning which doesn't affect the system